Jenkinsfile

```diff
@@ -0,0 +1,39 @@
+@Library("jenkins-library@main")
+
+import com.logicalclocks.jenkins.k8s.ImageBuilder
+
+properties([
+  parameters([
+    choice(name: 'image', choices: ['all', 'sklearnserver'],  description: 'Which docker image to build'),
```

Comment:
Not sure what 'all' means here since there is just one image atm?

Reply:
We can add xgbserver later.